### ISSUE TITLE
tmap_leaflet fails with valid static maps

### ISSUE THREAD

**Issue by walts** [mtennekes/tmap#404]
I'm using tmap 2.3-2 with sf 0.9-0.  I have maps which plot in static mode but fail in view mode and with tmap_leaflet with "Error: Unable to set the projection to +proj=longlat +datum=WGS84 +no_defs" and the warning In `$.crs`(gm$shape.master_crs, "proj4string") :
  CRS uses proj4string, which is deprecated."  Did the sf update break this?  Thanks.

**Comment by Nowosad** [comment]
@walts have you tried to install the development version of **tmap**:

```
library(devtools)
install_github("mtennekes/tmaptools")
install_github("mtennekes/tmap")
```
?

**Comment by walts** [comment]
Jakub,

 

Thanks for the quick response.

 

I installed the development version using code below. This skipped the install of tmaptools 3.0 but failed on tmap with the message “namespace 'leafem' 0.0.1 is being loaded, but >= 0.0.7 is required”.  I reinstalled leafem from CRAN which is still 0.0.1 and reinstalled TMAP 2.3-0.  This worked with the warning message “replacing previous import ‘sf::st_make_valid’ by ‘lwgeom::st_make_valid’ when loading ‘tmap’ “.  
 
I had reinstalled tmap when I first encountered the problem so not sure what fixed it.
 
Thanks for working with me on this.
 
Walter
